Source/Core/DolphinQt/Debugger/CodeWidget.cpp

```diff
@@ -172,6 +187,9 @@ void CodeWidget::ConnectWidgets()
 
 void CodeWidget::OnSearchAddress()
 {
+  if (m_search_address->text().size() != 8)
```

Comment:
Wouldn't this make addresses like `0x12345678` incorrect?

Reply:
Hmm, yes. I suppose I should allow 0x even though it's redundant